doc/glossary.rst

```diff
@@ -15,44 +15,67 @@ Frame:
   the term "frame" is used to indicate
   instances where the pose may not yet be known.
 
-Camera Pose:
-  In TeleSculptor, a camera pose
-  (often shortened to "camera")
+Camera:
+  In TeleSculptor, a camera
   refers primarily to the model
-  which describes the properties of a camera pose,
-  including attributes such as focal length
-  and world position and orientation.
+  which describes the intrinsic and extrinsic parameters of a camera
+  at a given frame.
+  Intrinsic parameters include focal length and lens distortion.
+  Extrinsic parameters are the camera pose |--| orientation and position.
   A frame may have at most one camera pose,
-  which is unique to that frame
-  (that is, camera poses are never shared)
-  In the GUI, cameras are represented as frustums.
+  which is unique to that frame.
+  While extrinsic parameters are never shared across frames,
+  intrinsic parameters may or may not be share depending on the algorithm
+  that produced them.
+
+  In the GUI, cameras are represented as rectangular pyramids.
+  The peak of the pyramid is the camera center of projection.
+  The rectangle base is proportional to the image aspect ratio,
+  and a triangle attached to the base indicates
+  which direction is "up" in the image.
+  The pyramid visualizes the field of view of the camera.
 
-  At present, TeleSculptor assumes that all poses
-  have the same camera *intrinsics*;
-  that is, that all frame images are captured
-  using the same physical camera.
-  Note that this includes the lens' focal length.
 
```

Comment:
Okay (when I wrote this, I hadn't realized there *is* an option to not force shared intrinsics), but please squash the consecutive blank lines.